src/Renderer.cpp

```
@@ -31,50 +24,40 @@ Math::Vector3D Raytracer::Renderer::rayColor(Ray &r, const IShape &shape,
-  Raytracer::Plane ground(
-      Math::Point3D(0, 1, 0),
-      Math::Vector3D(0, -1, 0),
-      Math::Vector3D(0.8, 0.8, 0.8)
-  );
-  group.addShape(std::make_shared<Raytracer::Plane>(ground));
-  
-  std::ofstream file(path);
-
-  writeHeader(file);
-
+void Raytracer::Renderer::renderToBuffer(std::vector<sf::Color> &framebuffer,
```

Cppcheck (2.10-2)

[style] The function 'renderToBuffer' is never used.

(unusedFunction)